### PR TITLE
add more ci resources and utilization

### PR DESCRIPTION
**Pull request by LHT129**
- from medium to medium+
- add more parallel for test and add logger

### CODE REVIEW

**Review by wxyucs — APPROVED** [review]
lgtm

File: scripts/test_asan_bg.sh [hunk 1]
```diff
@@ -2,37 +2,51 @@
...
-    break
+    echo ${logger_files[${index}]} "failed"
+    cat ${logger_files[${index}]}
+  else
+    echo ${logger_files[${index}]} "success"
   fi
+  ((index+=1))
 done
 
+rm -rf ./log
```
Comment (jiaweizone) [inline scripts/test_asan_bg.sh:48]: why need remove log ?

Comment (LHT129) [reply scripts/test_asan_bg.sh:48]: the scripts works on ci, I think remove or not have no difference.

**Review by jiaweizone — APPROVED** [review]
LGTM

**Pull request merged**